classy/gui.py

```diff
@@ -5,10 +5,14 @@
 import classy
 from classy.log import logger
 
-import pyqtgraph as pg
-
-# from pyqtgraph.Qt import QtCore, QtWidgets
-from pyqtgraph.Qt import QtWidgets, QtGui
+try:
+    import pyqtgraph as pg
+    from pyqtgraph.Qt import QtWidgets
+except (Exception, ModuleNotFoundError) as e:
```

Comment:
FWIW ModuleNotFoundError is a subclass of Exception so it doesn't have to be caught explicitly if Exception is also listed.

```suggestion
except Exception as e:
```